x/crosschain/keeper/msg_server_add_to_intx_tracker.go

```diff
@@ -21,8 +22,7 @@ func (k msgServer) AddToInTxTracker(goCtx context.Context, msg *types.MsgAddToIn
 		return nil, observertypes.ErrSupportedChains
 	}
 
-	adminPolicyAccount := k.zetaObserverKeeper.GetParams(ctx).GetAdminPolicyAccount(observertypes.Policy_Type_group1)
-	isAdmin := msg.Creator == adminPolicyAccount
+	isAdmin := k.GetAuthorityKeeper().IsAuthorized(ctx, msg.Creator, authoritytypes.PolicyType_groupEmergency)
```

Comment:
It's not an emergency. I think the naming `groupEmergency` just doesn't well describe the scenario. I don't have a name in my mind yet, maybe 'groupOperation'?

Reply:
It's not an emergency. I think the naming groupEmergency just doesn't well describe the scenario. I don't have a name in my mind yet, maybe 'groupOperation'?

Reply:
We used to have group 1 and 2 to group actions depending on required signer but needed to be renamed to be clearer.
In general, action requiring 1 vote are for emergency situation, this is why the group is named emergency. For this one this is quite an exception since it is not an emergency properly said but should not require more than 1 vote.

For some of the exception I think it is fine to keep this naming instead of creating a whole new group.

Group Operation is going to be created but it will require more than 1 vote

In vote numbers: emergency < operation < admin

Reply:
Also adding trackers can be seen as a "low-urgency" emergency as the action can potentially unblock the network